### ISSUE TITLE
starter hello-world babel helper-compilation-targets error

### ISSUE THREAD

**Issue by comoser** [gatsbyjs/gatsby#22145]
## Description

Following the tutorial to create a new hello-world gatsby application on page: https://www.gatsbyjs.org/tutorial/part-zero/ we are faced with a compilation error when running `gatsby develop` IF we are on node v.13.10+

### Steps to reproduce

On node v13.10.x when using `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world` as in the [tutorial](https://www.gatsbyjs.org/tutorial/part-zero/) of the official Gatsby page the following dependency `@babel/helper-compilation-targets` is locked in `package-lock.json` with the version `7.8.3`.

When trying to follow the run instructions we do:
`cd hello-world`
`gatsby develop`

and we are met with the following error:
```
ERROR #98123  WEBPACK

Generating SSR bundle failed

[BABEL] ~/gatsby-tests/test/.cache/develop-static-entry.js: No "exports" main resolved in ~/gatsby-tests/test/node_modules/@babel/helper-compilation-targets/package.json

File: .cache/develop-static-entry.js
```

### Expected result

The starter hello-world project should run with `gatsby develop` on localhost.

### Actual result

The starter hello-world project gives an error related to a babel helper plugin.

```
ERROR #98123  WEBPACK

Generating SSR bundle failed

[BABEL] ~/gatsby-tests/test/.cache/develop-static-entry.js: No "exports" main resolved in ~/gatsby-tests/test/node_modules/@babel/helper-compilation-targets/package.json

File: .cache/develop-static-entry.js
```

### Environment

```
System:
    OS: macOS Mojave 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 13.10.1 - /usr/local/bin/node
    Yarn: 1.15.2 - /usr/local/bin/yarn
    npm: 6.13.7 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 80.0.3987.132
    Firefox: 73.0.1
    Safari: 13.0.5
```

**Comment by comoser** [comment]
Will submit a PR for this fix

**Comment by comoser** [comment]
@gatsbyjs/core issue fixed on #22166
